### PR TITLE
Pip install

### PR DESCRIPTION
**Pull request by jburel**
Make gallery pip installable 
and rename the folder

To test:

```
$ mkdir test-gallery
$ cd test-gallery
$ virtualenv venvweb
$ source venvweb/bin/activate
(venvweb) $ pip install omego 
(venvweb) $ omego download --ice 3.6 --branch OMERO-DEV-merge-build py
(venvweb) $ mv OMERO.py[currentversion] OMERO.py
(venvweb) $ pip install -r OMERO.py/share/web/requirements-py27-trial.txt

(venvweb) $ pip install git+git://github.com/jburel/gallery.git@pip-install
(venvweb) $ OMERO.py/bin/omero config append omero.web.apps '"omero_gallery"'
(venvweb) $ OMERO.py/bin/omero web start
```
When testing locally, unset omerohome, if set,  to avoid surprises

cc @aleksandra-tarkowska @sbesson @joshmoore

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,30 +1,33 @@
...
+Install OMERO.web
 
+This app installs into the OMERO.web framework.
 
-Place the contents of this repository (in a directory named 'gallery') within a
-location on your $PYTHONPATH. Then add it to your installed web apps.
+::
 
-    $ cd folder/on/pythonpath/
+    $ pip install gallery
```
Comment (aleksandra-tarkowska) [inline README.rst]: omero-gallery

**Comment by aleksandra-tarkowska** [comment]
Thanks for this PR. I will fix the job and update config in MT to get that to CI on Monday.

**Comment by jburel** [comment]
@aleksandra-tarkowska: I have not renamed the repo yet, I was planning to have a second PR or a second round when this PR has been tested

Thanks for taking care of the job

**Comment by aleksandra-tarkowska** [comment]
I know. My comments are about pip package not the repo. It was agreed to use PyPi name with omero- prefix. Please check figure repo as that hasn't been renamed yet as well

**Comment by jburel** [comment]
@aleksandra-tarkowska: done

**Comment by sbesson** [comment]
@aleksandra-tarkowska @jburel: if we want to test the whole workflow including the Pypi upload with the new package name as discussed above, we could also consider using pypitest for the packaging/registration/upload step - see https://wiki.python.org/moin/TestPyPI

**Comment by jburel** [comment]
@sbesson: good idea
I will look into it later on. (probably not today!)

**Comment by jburel** [comment]
Tested on testpypi: https://testpypi.python.org/pypi/omero-gallery/2.0.0
installed from repo
``
pip install -i https://testpypi.python.org/pypi omero-gallery
Collecting omero-gallery
  Downloading https://testpypi.python.org/packages/70/fa/9c44dfefe5e1ebc57ce0961117eb850c8ca5ad70d53bd14dccf55a184965/omero-gallery-2.0.0.tar.gz (225kB)
    100% |████████████████████████████████| 229kB 1.4MB/s 
Building wheels for collected packages: omero-gallery
  Running setup.py bdist_wheel for omero-gallery ... done
...
Successfully built omero-gallery
Installing collected packages: omero-gallery
Successfully installed omero-gallery-2.0.0
``

**Comment by jburel** [comment]
**Important:** your credentials for pypi will not work for testpypi
you need to register again.

**Comment by jburel** [comment]
created a 2.0.1 version on testpypi to test 696e7b6
see https://testpypi.python.org/pypi?:action=display&name=omero-gallery&version=2.0.1

**Comment by jburel** [comment]
@sbesson: I also think we should add a section to https://www.openmicroscopy.org/site/support/contributing/tools-maintenance.html
about how to test package using testpypi. I will draft something

**Comment by sbesson** [comment]
@jburel makes senses. My feeling is that Pypi test is useful in 2 cases: either when initiating a new Python package or when training a new Python maintainer. If we review the content of the page above, we might want to promote it more generally as the OME Python packages maintenance page (now we have more than development tools).

**Comment by aleksandra-tarkowska** [comment]
all looks good, works as expected tested via https://github.com/openmicroscopy/management_tools/pull/203/commits/fe672b7f6f4f5bffd601c757b407fe91196d811d available on http://web-dev-integration.openmicroscopy.org/gallery/

either we can merge this PR or rename this repo and I will retest and then we can merge this PR and push to pypi, up to you

**Pull request merged**